tests/environment.py

```diff
@@ -365,24 +374,32 @@ class ALEntityPrplWrt(ALEntity):
     def __init__(self, name: str, device: None, is_controller: bool = False):
         self.device = device
         self.name = name
-        self.bridge_name = 'br-lan'
 
         if is_controller:
             self.config_file_name = '/opt/prplmesh/config/beerocks_controller.conf'
         else:
             self.config_file_name = '/opt/prplmesh/config/beerocks_agent.conf'
 
-        ucc_port = self.command(("grep \"ucc_listener_port\" {} "
-                                "| cut -d'=' -f2 | cut -d\" \" -f 1").format(self.config_file_name))
-
-        device_ip_output = self.command(
-                "ip -f inet addr show {} | head -n 2".format(self.bridge_name))
-        device_ip = re.search(
-            r'inet (?P<ip>[0-9.]+)', device_ip_output.decode('utf-8')).group('ip')
-        self.log_folder = self.command(
-            "grep log_files_path {} | cut -d=\'=\' -f2".format(self.config_file_name))
-
-        ucc_socket = UCCSocket(device_ip, ucc_port)
+        ucc_port_raw = self.command("grep \"ucc_listener_port\" {}".format(self.config_file_name))
+        ucc_port = int(re.search(r'ucc_listener_port=(?P<port>[0-9]+)',
+                                 ucc_port_raw).group('port'))
+        bridge_name_raw = self.command("grep \"bridge_iface\" {}".format(self.config_file_name))
+        self.bridge_name = re.search(r'bridge_iface=(?P<bridge>.+)\r\n',
+                                     bridge_name_raw).group('bridge')
+
+        # Multiple IPs may be set to same interface. We are interested in the last one, which is set
```

Comment:
This will also no longer be true if we can use the host network for the Docker container.